### PR TITLE
First updates

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+inherit_from: .rubocop_todo.yml
+
+require:
+  - rubocop-performance
+  - rubocop-rails
+  - rubocop-minitest
+
+AllCops:
+  TargetRubyVersion: 2.5
+  TargetRailsVersion: 6.0
```
Comment (ekohl) [inline .rubocop.yml]: I noticed this in the template already: don't we target 2.7 & 6.1 now?

File: foreman_theme_satellite.gemspec [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
-  s.name        = "foreman_theme_satellite"
+  s.name        = 'foreman_theme_satellite'
   s.version     = ForemanThemeSatellite::VERSION
-  s.date        = Date.today.to_s
-  s.authors     = ["Alon Goldboim, Shimon Stein"]
-  s.email       = ["agoldboi@redhat.com"]
+  s.metadata    = { "is_foreman_plugin" => "true" }
+  s.license     = 'GPL-3.0'
+  s.authors     = ['Alon Goldboim, Shimon Stein']
+  s.email       = ['agoldboi@redhat.com']
```
Comment (ekohl) [inline foreman_theme_satellite.gemspec]: Do we have a better email address?

File: foreman_theme_satellite.gemspec [hunk 3]
```diff
@@ -1,21 +1,27 @@
...
   # also update locale/gemspec.rb
-  s.description = "Theme changes for Satellite 6."
+  s.description = 'Theme changes for Satellite 6.'
   s.files = Dir['{app,config,db,lib,locale,webpack}/**/*'] +
             ['LICENSE', 'Rakefile', 'README.md'] +
             ['package.json']
   s.files -= Dir['**/*.orig']
-  s.test_files = Dir['test/**/*']
-  s.test_files -= Dir['test/**/*.orig']
+  s.test_files = Dir['test/**/*'] + Dir['webpack/**/__tests__/*.js']
```
Comment (ekohl) [inline foreman_theme_satellite.gemspec]: https://guides.rubygems.org/specification-reference/ doesn't mention `test_files` anymore. RuboCop's `Gemspec/DeprecatedAttributeAssignment` cop should flag this.

File: .rubocop_todo.yml [hunk 4]
```diff
@@ -0,0 +1,534 @@
...
+# Configuration parameters: TreatCommentsAsGroupSeparators, ConsiderPunctuation, Include.
+# Include: **/*.gemspec
+Gemspec/OrderedDependencies:
+  Exclude:
+    - 'foreman_theme_satellite.gemspec'
+
+# Offense count: 1
+# Configuration parameters: Severity, Include.
+# Include: **/*.gemspec
+Gemspec/RequiredRubyVersion:
```
Comment (ekohl) [inline .rubocop_todo.yml]: Can we fix this instead?

File: .rubocop_todo.yml [hunk 5]
```diff
@@ -0,0 +1,534 @@
...
+  Exclude:
+    - 'Gemfile'
+
+# Offense count: 2
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: TreatCommentsAsGroupSeparators, ConsiderPunctuation, Include.
+# Include: **/*.gemspec
+Gemspec/OrderedDependencies:
+  Exclude:
+    - 'foreman_theme_satellite.gemspec'
```
Comment (ekohl) [inline .rubocop_todo.yml]: I'd also prefer to fix these.

File: .rubocop.yml (hunk 1 above)
Comment (evgeni) [reply .rubocop.yml]: technically, yes.
updated and opened https://github.com/theforeman/foreman_plugin_template/pull/48

File: .rubocop_todo.yml (hunk 4 above)
Comment (evgeni) [reply .rubocop_todo.yml]: done

File: .rubocop_todo.yml (hunk 5 above)
Comment (evgeni) [reply .rubocop_todo.yml]: done

File: foreman_theme_satellite.gemspec (hunk 3 above)
Comment (evgeni) [reply foreman_theme_satellite.gemspec]: indeed, but that is not enabled in our config.

File: foreman_theme_satellite.gemspec (hunk 2 above)
Comment (evgeni) [reply foreman_theme_satellite.gemspec]: It comes down to the definition of "we".
In Foreman, we can (and do in some cases) use `dev@community`.
But we are not in Foreman here, so maybe not?

We could ask @ShimShtein nicely if he wants to be listed here for now

**Review by ehelms — APPROVED** [review]

**Comment by evgeni** [comment]
~~The "fix provisioning links" might not be required (talked to our Docs team and they say the link change was unintentional), so waiting on the result of that with the merge~~

they are required

**Comment by ekohl** [comment]
> they are required

 Came here to link https://github.com/theforeman/foreman-documentation/pull/2542#issuecomment-1771084480 which concluded they were needed.

**Pull request merged**